frontend/pages/hosts/details/OSSettingsModal/OSSettingsTable/OSSettingsTableConfig.tsx

```
@@ -103,7 +103,7 @@ const generateTableConfig = (
         const isAndroidCertificate =
           platform === "android" &&
           cellProps.row.original.profile_uuid ===
-            FLEET_ANDROID_CERTIFICATE_TEMPLATE_PROFILE_ID;
+          FLEET_ANDROID_CERTIFICATE_TEMPLATE_PROFILE_ID;
```

ESLint

[error] Insert `··`

(prettier/prettier)